### ISSUE TITLE
Notifying/inviting people — privacy/security problem

### ISSUE THREAD

**Issue by madduck** [loomio/loomio#7571]
One of the members in a new group didn't enter anything but their first name into their Loomio profile. When I tried to invite them into a thread, I was being offered people from other groups I am in:

![image](https://user-images.githubusercontent.com/195073/117905419-3d641e00-b327-11eb-99c1-060560bc524d.png)

I assume there are other Dans on Loomio, so I am glad that there's a pre-filter in place, but this UI also presents a couple of problems:
1. I might accidentally invite an external person to a potentially sensitive thread.
2. If there is more than one "Dan", I have no way to actually figuring out which one is the right Dan.

Maybe specifying how I am "related" to each would help? Alternatively, a visual separation of members of the current group, and externals could help. It's great to be able to invite external people, but there ought to be a sanity check or two…

**Comment by robguthrie** [comment]
Good call. We should divide the list up into members and external to the group
